src/chart-review/encounters/encounters.component.tsx

```diff
@@ -1,42 +1,37 @@
 import React from "react";
-import { useParams, useLocation, Link } from "react-router-dom";
+import { useParams, useLocation, Link, useRouteMatch } from "react-router-dom";
 import ChartWidget from "../../ui-components/chart-widget/chart-widget.component";
 import ProgramsLevelTwo from "../../widgets/programs/programs-level-two.component";
 import ConditionsDetailedSummary from "../../widgets/conditions/conditions-detailed-summary.component";
 import { AllergyOverviewLevelTwo } from "../../widgets/allergies/allergy-card-level-two.component";
 import styles from "../../ui-components/chart-widget/chart-widget.css";
+import NotesOverview from "../../widgets/notes/notes-overview.component";
 
 function useQuery() {
   return new URLSearchParams(useLocation().search);
 }
 
 export default function Encounters(props: any) {
+  const match = useRouteMatch();
+
   const widgetConfig = {
     name: "encounters",
-    defaultTabIndex: 0,
+    defaultTabIndex: 1,
```

Comment:
what happens if multiple widgets have the same tab index?

Reply:
That's not possible.  In this case, I've moved  allergies/conditions/programs to the main nav. In this config, you only choose a specific main nav item. Each component that this refers to, if it has a tab, can also set it's default item. So for example, the Results widget can also identify which defaulit would like to display.